src/webui/api/torrentscontroller.cpp

```diff
@@ -368,6 +406,15 @@ void TorrentsController::infoAction()
 
         QVariantMap serializedTorrent = serialize(*torrent);
 
+        if (includeFiles)
+        {
+            const int filesCount = torrent->filesCount();
+            QList<int> fileIndexes;
+            fileIndexes.reserve(filesCount);
+            for (int i = 0; i < filesCount; ++i)
+                fileIndexes.append(i);
```

Comment:
I would move this logic to `getFiles()` so that it allows you to use empty list as a `fileIndexes` placeholder to return all files.